### ISSUE TITLE
Ruby 3 compatibility

### ISSUE THREAD

**Issue by jcoyne** [svenfuchs/rails-i18n#959]
This commit introduced code that is not ruby 3 compatible:
https://github.com/svenfuchs/rails-i18n/commit/f6cc43ec8cfc08d0db7a06ca8299d7834c517a05

```
  /opt/hostedtoolcache/Ruby/3.0.3/x64/lib/ruby/gems/3.0.0/bundler/gems/rails-i18n-751d497ed2a7/lib/rails_i18n/../../rails/ordinals/fr-CA.rb:2: syntax error, unexpected ':', expecting =>
    :"fr-CA": {
            ^
  /opt/hostedtoolcache/Ruby/3.0.3/x64/lib/ruby/gems/3.0.0/bundler/gems/rails-i18n-751d497ed2a7/lib/rails_i18n/../../rails/ordinals/fr-CA.rb:19: syntax error, unexpected '}', expecting end-of-input
```

**Comment by jcoyne** [comment]
It looks like #958 would fix this issue.

**Comment by tagliala** [comment]
I think this has been fixed in 7.0.1
